lib/BC/Util.cpp

```diff
@@ -1205,17 +1205,6 @@ MoveConstantIntoModule(llvm::Constant *c, llvm::Module *dest_module,
         moved_c = ret;
         return ret;
       }
-      case llvm::Instruction::Select: {
```

Comment:
why are we removing this case?

Reply:
llvm 17 weakened constant expressions significantly, you cant create a select anymore